middleware/requestid/requestid_test.go

```diff
@@ -64,12 +64,12 @@ func Test_RequestID_Locals(t *testing.T) {
 
 	var ctxVal string
 
-	app.Use(func (c *fiber.Ctx) error {
+	app.Use(func(c *fiber.Ctx) error {
```

Comment:
Please remove the requestid mw related stuff from this PR. You can make another one of course:smile: